src/app/globals.css

```diff
@@ -280,7 +280,8 @@ body {
       display: inline-block;
       max-width: var(--max-key-length);
       white-space: nowrap;
-      overflow-x: auto;
```

Comment:
We should not remove the scrollbar until there is a tooltip to display values.

Reply:
Is there a reason that it uses to set html to dangerouslySetInnerHTML in table mode? This seems to make it hard to integrate a component, isn't it?

Reply:
Yes, but it's much faster to generate DOM in web worker than use react in main thread. If you are interesting, you can try it with [big files](https://gist.github.com/loggerhead/8540ee26b8512e2ee314f7050461044e).